vips/vip-498/bscmainnet-2.ts

```diff
@@ -0,0 +1,254 @@
+import { BigNumber } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { LzChainId, ProposalType } from "src/types";
+import { makeProposal } from "src/utils";
+
+const { ethereum } = NETWORK_ADDRESSES;
+
+export const VTOKEN_RECEIVER = "0x3e8734ec146c981e3ed1f6b582d447dde701d90c";
+export const COMPTROLLER_CORE = "0x687a01ecF6d3907658f7A7c714749fAC32336D1B";
+export const sUSDe = "0x9D39A5DE30e57443BfF2A8307A4256c8797A3497";
+export const USDe = "0x4c9EDD5852cd905f086C759E8383e09bff1E68B3";
+export const VsUSDe_CORE = "0xa836ce315b7A6Bb19397Ee996551659B1D92298e";
+export const VUSDe_CORE = "0xa0EE2bAA024cC3AA1BC9395522D07B7970Ca75b3";
+export const sUSDe_INITIAL_SUPPLY = parseUnits("10000", 18);
+export const USDe_INITIAL_SUPPLY = parseUnits("10000", 18);
+export const convertAmountToVTokens = (amount: BigNumber, exchangeRate: BigNumber) => {
+  const EXP_SCALE = parseUnits("1", 18);
+  return amount.mul(EXP_SCALE).div(exchangeRate);
+};
+
+const sUSDe_ERC4626ORACLE = "0x67841858BCCA8dF50B962d6A314722a6AEC0970e";
+const REDSTONE_USDe_FEED = "0xbC5FBcf58CeAEa19D523aBc76515b9AEFb5cfd58";
+const CHAINLINK_USDe_FEED = "0xa569d910839Ae8865Da8F8e70FfFb0cBA869F961";
+const STALE_PERIOD_YEAR = 60 * 60 * 24 * 365; // 1 Year
```

Comment:
```suggestion
const STALE_PERIOD_YEAR = 60 * 60 * 26; // The heartbeat is 24 hours
```

Reply:
@chechu the stale period is not working at 26 hours . it's working when it is 3 * 26 hours .